internal/service/alarms/internal/alertmanager/alertmanager.yaml

```diff
@@ -0,0 +1,30 @@
+---
+global:
+  resolve_timeout: 5m
+route:
+  group_wait: 30s
+  group_interval: 1m
+  repeat_interval: 4h
+  # Default receiver - Alerts without cluster identifier label will still be sent to this receiver
+  receiver: oran_alarm_receiver
+  routes:
+  # Route to skip alerts
+  - receiver: "null"
+    matchers:
+      # Always firing alert to verify alertmanager is working
+      - Watchdog
```

Comment:
I think skipping `Watchdog` is fine but just want to make sure @browsell is aware of this since we initially sort of agreed to collect all the alerts regardless of it being a noise.